R/tidyeval.R

```diff
@@ -55,35 +55,48 @@
 #' x <- 1
 #' partial_eval(quote(x ^ y))
 partial_eval <- function(call, vars = character(), env = caller_env()) {
-  if (is_null(call)) {
-    NULL
-  } else if (is_atomic(call) || blob::is_blob(call)) {
+  # corresponds to `dt_squash()`
+  # TODO change `vars` to `data`
```

Comment:
I think it's worth making this change; I doubt there are many existing uses, and we could provide an informative deprecation based on either the argument name or the type (string vs. data frame or whatever it actually is).